docs/how-to/hyperparameter-tuning.txt

```diff
@@ -76,15 +76,16 @@ use in each trial.  Every searcher is configured with the name of the
 validation metric to optimize (via the ``metric`` field), in addition to
 other searcher-specific options. For example, the ``adaptive_simple``
 searcher implements adaptive downsampling, and is configured with the
-maximum number of trials to run and the maximum number of training steps
-allowed per trial:
+maximum number of trials to run and the maximum training length allowed
+per trial:
```

Comment:
Link to training units def here?

Reply:
Makes sense. I didn't want to go crazy linking it everywhere if I wasn't 100% sure it needed it.